sx-request.el

```diff
@@ -164,7 +201,8 @@ the main content of the response is returned."
                      sx-request-remaining-api-requests-message-threshold)
               (sx-message "%d API requests remaining"
                           sx-request-remaining-api-requests))
-            (sx-encoding-clean-content-deep .items)))))))
+            (funcall (or process-function #'sx-request-response-get-items)
+                       response)))))))
```

Comment:
Maybe it's just the mobile app I'm reading this on, but I think indentation is off here.